src/modules/projects/components/ProjectForm/ProjectForm.tsx

```diff
@@ -0,0 +1,153 @@
+// import React from 'react'
+
+import { useEffect } from 'react';
+import { useForm } from 'react-hook-form';
+import { MdKeyboardArrowLeft } from 'react-icons/md';
+import { Link, useNavigate, useParams } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import {
+  axiosInstance,
+  PROJECTS_URLS,
+} from '../../../../services/apisUrls/apisUrls';
+import { GetRequiredMessage } from '../../../../services/validation/validation';
+
+interface ProjectFormData {
+  title: string;
+  description: string;
+}
+interface GetProjectByIdResponse {
+  data: any;
```

Comment:
We need to add the actual type for the data to get auto completion and errors if accessing incorrect value.